index.html

```diff
@@ -543,7 +543,7 @@ <h4 id="return-values">Return Value</h4>
     <p>A request initiated by sendBeacon is subject to following properties:</p>
     <ul>
       <li>If the payload is one of <code>application/x-www-form-urlencoded</code>, <code>multipart/form-data</code>, or <code>text/plain</code>, then the request is a simple request that does not require an additional CORS-preflight; same as scripted form-post or XHR/fetch.</li>
```

Comment:
It's much more similar to a form that uses the POST method (which not incidentally has the same restrictions on `Content-Type` and such).

I don't see how your "Otherwise" statement works. It's never similar to XHR/fetch when the `Content-Type` is safelisted due to redirects. XHR and fetch require any redirects to have CORS headers.